build.sbt

```diff
@@ -20,6 +20,10 @@ lazy val commonSettings = Seq(
     "-language:experimental.macros",
     "-feature"
   ),
+  bintrayOrganization := Some("bintray"),
+  bintrayRepository := "jcenter",

```

Comment:
I expected to see azavea/geotrellis here. Are we actually able to publish here and if so what does it mean?

Reply:
I'm actually not sure. `geotrellis-gdal` isn't under azavea on BinTray https://bintray.com/bintray/jcenter/com.azavea.geotrellis%3Ageotrellis-gdal_2.11 So I wasn't sure if we should be publishing to that or to something else.